### DISCUSSION TITLE
Readme need correction with regard to cert-manager version

### DISCUSSION THREAD

**Discussion by iamraj007** [rabbitmq/messaging-topology-operator#874] [Q&A] [answered]
## Need to update Readme
Readme section mention if you have cert-manager 1.2.0 or above... compatibility of messaging-topology-operator-with-certmanager works very well but  cert-manager highest version is  [1.15](https://cert-manager.io/docs/releases/#upcoming-releases) so is the documentation not correct or am i missing something?  

below is extract from ReadMe:
If you have [cert-manager](https://cert-manager.io/docs/installation/kubernetes/) 1.2.0 or above installed in your k8s cluster, and kubectl configured to access your running k8s cluster, you can then run the following command to install the Messaging Topology Operator

**Comment by Zerpet** [comment, marked as answer]
The minimum supported version of cert-manager is 1.2.0. It should also work with the latest 1.15 version too. The readme states:

> If you have [cert-manager](https://cert-manager.io/docs/installation/kubernetes/) 1.2.0 or **above**

Noting that any cert-manager version  >= 1.2.0 is compatible.

**Reply by MirahImage** [reply]
To be extra clear, 15 > 2
